PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -71,6 +71,30 @@ opts_AddOptional '--processing-mode' 'ProcessingMode' 'HCPStyleData|LegacyStyleD
 opts_AddOptional '--structural-qc' 'QCMode' 'yes|no|only' "whether to run structural QC, default 'yes'" 'yes'
 opts_AddOptional '--use-ind-mean' 'UseIndMean' 'YES or NO' "whether to use the mean of the subject's myelin map as reference map's myelin map mean, defaults to 'YES'" 'YES'
 
+#Longitudinal_mode modes:
```

Comment:
I wonder if some of these comments should actually get printed in the usage.  Is that possible @coalsont?

Reply:
You can put newlines in the description string, quotes work for that.  See here:

https://github.com/Washington-University/HCPpipelines/blob/9cfa37a378dff26fd5aa4fd2aad24d0136830f0f/global/scripts/RSNregression.sh#L56-L59

Just make sure there aren't any tab characters, the word wrapping code doesn't try to predict their behavior.

Reply:
done.